src/Apps/About/AboutApp.tsx

```diff
@@ -84,6 +97,162 @@ export const AboutApp: React.FC<React.PropsWithChildren<unknown>> = () => {
       <Spacer y={6} />
 
       <AboutArtworksRailQueryRenderer />
+
+      <StructuredData
+        schemaData={{
+          "@context": "https://schema.org",
+          "@type": "WebApplication",
+          name: "Artsy",
+          url: "https://www.artsy.net",
+          applicationCategory: "LifestyleApplication",
+          operatingSystem: "Web",
+          browserRequirements: "Requires JavaScript",
+          description: DESCRIPTION,
+          downloadUrl: [
+            DOWNLOAD_APP_URLS[Device.iPhone],
+            DOWNLOAD_APP_URLS[Device.Android],
+          ],
+          offers: {
+            "@type": "Offer",
+            price: "0",
+            priceCurrency: "USD",
+            category: "free",
+          },
+          inLanguage: "en",
+          author: {
+            "@type": "Organization",
+            name: "Artsy",
+            url: "https://www.artsy.net",
+          },
+          hasPart: [
+            {
+              "@type": "MobileApplication",
+              name: "Artsy",
+              operatingSystem: "iOS 16.6 or later",
+              applicationCategory: "LifestyleApplication",
+              downloadUrl: DOWNLOAD_APP_URLS[Device.iPhone],
+              installUrl: DOWNLOAD_APP_URLS[Device.iPhone],
+              contentRating: "17+",
+              offers: {
+                "@type": "Offer",
+                price: "0",
+                priceCurrency: "USD",
+                category: "free",
+              },
+              aggregateRating: {
+                "@type": "AggregateRating",
+                ratingValue: "4.8",
+                reviewCount: "5500",
+              },
+            },
+            {
+              "@type": "MobileApplication",
+              name: "Artsy",
+              applicationCategory: "LifestyleApplication",
+              downloadUrl: DOWNLOAD_APP_URLS[Device.Android],
+              installUrl: DOWNLOAD_APP_URLS[Device.Android],
+              contentRating: "Teen",
+              offers: {
+                "@type": "Offer",
+                price: "0",
+                priceCurrency: "USD",
+                category: "free",
+              },
+              aggregateRating: {
+                "@type": "AggregateRating",
+                ratingValue: "4.5",
+                reviewCount: "1230",
+              },
+            },
+          ],
+        }}
+      />
+
+      <StructuredData
```

Comment:
I'm not sure why we'd need to have 1 object for the application and another for the organization. Would we want to do this across all pages? @dzucconi

Reply:
It's fine to have multiple objects on a page. Unclear actually if we'd want it on every page, I suspect not but that's more a question for Botify. We can validate the markup as well. Example: https://validator.schema.org/#url=https%3A%2F%2Fwww.artsy.net%2Fartwork%2Fdamon-zucconi-security